src/textual/widget.py

```diff
@@ -1143,25 +1137,21 @@ def _get_box_model(
             styles_height = styles.height
             # Explicit height set
             content_height = styles_height.resolve(
-                sizing_container - styles.margin.totals, viewport, height_fraction
+                container - styles.margin.totals, viewport, height_fraction
             )
-            if is_border_box and styles_height.excludes_border:
```

Comment:
Not sure what I was thinking re `excludes_border`. But it doesn't appear to be required.

Reply:
I agree it doesn't appear to be required at all.